gramps/plugins/drawreport/ancestortree.py

```diff
@@ -74,6 +75,11 @@
 
 LVL_GEN, LVL_INDX, LVL_Y = range(3)
 
+_BOXCOLOR_DEFAULT = (255,255,255) # default dark
```

Comment:
Can you please use the users default colors already defined in the Colors preference tab see

https://gramps-project.org/wiki/index.php/Gramps_5.2_Wiki_Manual_-_Settings#Colors